rviz_default_plugins/src/rviz_default_plugins/displays/wrench/wrench_display.cpp

```diff
@@ -127,11 +132,31 @@ bool validateFloats(const geometry_msgs::msg::WrenchStamped & msg)
 
 void WrenchDisplay::processMessage(geometry_msgs::msg::WrenchStamped::ConstSharedPtr msg)
 {
-  if (!validateFloats(*msg)) {
-    setStatus(
-      rviz_common::properties::StatusProperty::Error, "Topic",
-      "Message contained invalid floating point values (nans or infs)");
-    return;
+  auto adjusted_msg = std::make_shared<geometry_msgs::msg::WrenchStamped>();
+  bool accept_NaN = accept_nan_values_->getBool();
+
+  if (!accept_NaN) {
```

Comment:
Let's use all small letters in variable names.
```suggestion
  if (!accept_nan) {
```